### DISCUSSION TITLE
CALENDAR: Excluded dates have the same color as dates outside of the month, which is confusing

### DISCUSSION THREAD

**Discussion by Surm4** [mantinedev/mantine#6594] [Feedback]
The excluded dates (depicted in the image below from 1st to 25th of July) have the same color as the dates outside of the month (depicted in the image below from 1st to 4th of Aug.) . This is confusing because, although they share the same color, they serve different purposes and have different behaviors:

- Excluded dates are not selectable.
- Dates outside of the month are selectable.

The default behaviour of matine, suggests that they might be treated the same way, which leads to confusion.

![image](https://github.com/user-attachments/assets/8b92e546-57e3-4299-af19-ed38f03e4e7f)

**Comment by rtivital** [comment]
You can change styles of outside dates using `data-outside` attribute with Styles API.
Outside dates have hover styles by default, I consider this to be enough to distinguish them from disabled dates.

**Reply by Surm4** [reply]
@rtivital Yes, I know I can do that, however it's not a request for me, but for a library. Wouldn't it be better with intuitive UI by default?

Regarding "data-outside" - I think it's still not possible to style it like "inside" dates because mantine is missing information whether it's weekend for outside dates.

So can you just add data-weekend="true" for these dates also?
![image](https://github.com/user-attachments/assets/0a4001e1-9efe-4bf7-8d4a-01373b800cf5)
